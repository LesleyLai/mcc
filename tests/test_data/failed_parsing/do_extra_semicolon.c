int main(void) {
  do {
    int a;
  }; while(1);
  return 0;
}
