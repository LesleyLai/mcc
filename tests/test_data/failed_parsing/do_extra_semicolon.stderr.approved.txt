{{filename}}:4:4: Error: expect "while"
4 |   }; while(1);
  |    ^

